timeseries/src/autogluon/timeseries/dataset/ts_dataframe.py

```diff
@@ -525,3 +526,16 @@ def get_reindexed_view(self, freq: str = "S") -> TimeSeriesDataFrame:
         df_view._cached_freq = freq
 
         return df_view
+
+    def hash_each_item(self) -> pd.Series:
+        """Hash each time series in the dataset to a 32-character hex string.
+
+        Hash is computed based on the timestamps and values of the time series (item_id is ignored).
```

Comment:
Yes, that's intended since both these items should get the exact same predictions. We don't use any local models that incorporate static features now, and I don't think we intend to add any such models in the future, but we can revisit the hashing logic in case we ever do